### PR TITLE
added closing parens in AuthorManager. Added the background color.

### PR DESCRIPTION
**Pull request by pracinejr**

### CODE REVIEW

File: TabloidCLI/Program.cs [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 namespace TabloidCLI
 {
     class Program
     {
         static void Main(string[] args)
         {
+
+            Console.BackgroundColor = ConsoleColor.Cyan;
+            Console.Clear();
+            Console.ForegroundColor = ConsoleColor.DarkMagenta;
```
Comment (joshdbarton) [inline TabloidCLI/Program.cs]: This is a terrifying color combination.  Well done.

Comment (hcrudge) [reply TabloidCLI/Program.cs]: Looks good!!

**Comment by sxiong0519** [comment]
💯

**Comment by hcrudge** [comment]
Looks good!!

**Pull request merged**